lib/spack/docs/build_systems/makefilepackage.rst

```diff
@@ -0,0 +1,295 @@
+.. _makefilepackage:
+
+---------------
+MakefilePackage
+---------------
+
+The most primitive build system a package can use is a plain Makefile.
```

Comment:
Not true.  The most primitive is a script you have to edit.  Believe me, it still exists...

Reply:
Fair enough, although I'm not sure if that's considered a "build system" or not.